From cab7934e222f34401569119504227fab1c169dd3 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Wed, 17 Mar 2021 15:25:24 +0000
Subject: [PATCH] GCC: fix template specialization in
 ToV8Traits<IDLArray<T>>::ToV8

GCC complains that explicit specialization in non-namespace scope
is happening for ToV8Traits<IDLArray<T>>::ToV8. However, the
specialization isn't needed here so remove it.
---
 third_party/blink/renderer/bindings/core/v8/to_v8_traits.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/third_party/blink/renderer/bindings/core/v8/to_v8_traits.h b/third_party/blink/renderer/bindings/core/v8/to_v8_traits.h
index 6c79433..0e1f469 100644
--- a/third_party/blink/renderer/bindings/core/v8/to_v8_traits.h
+++ b/third_party/blink/renderer/bindings/core/v8/to_v8_traits.h
@@ -683,7 +683,6 @@ struct ToV8Traits<IDLArray<T>> {
   }
 
   // TODO(crbug.com/1185046): Remove this overload.
-  template <>
   static v8::MaybeLocal<v8::Value> ToV8(
       ScriptState* script_state,
       const Vector<v8::Local<v8::Value>>& value) WARN_UNUSED_RESULT {
-- 
2.26.2

